lib/connection.ts

```diff
@@ -365,6 +386,15 @@ export class Connection extends Entity {
     });
   }
 
+  /**
+   * **It is the synchronous version of `close` where the user can call `closeSync` and not**
+   * **worry about errors caused while closing the connection (fire and forget).**
+   * Closes the amqp connection.
+   */
+  closeSync(): void {
+    this.close().catch(() => { /** */ });
```

Comment:
If `closeSync` was the last call the user made in their program, the program wont quit until `close()` actually completes. So even though the name says "sync", it doesnt give any benefits to the user i.e the user might as well call `this.close()` and not await on it

Reply:
That is true. But, we need to add handlers for `*_error`. If we don't do that the error bubbles up from link to session to connection and then eventually turns into a general `Error` which can crash the app. 

> the user might as well call this.close() and not await on it

That is true. Will remove it.